### DISCUSSION TITLE
Using Bleachbit on Raspberry Pi OS

### DISCUSSION THREAD

**Discussion by 0rezer0** [bleachbit/bleachbit#1680] [General]
I've been a long time fan and user of Bleahbit, it gets installed on all of my machines. Lately I've been using Raspberry Pi's, from the 4 and now the 5 running Raspberry Pi OS based on Debian. I was wondering if anyone else does the same and has any tips for using Bleachbit in this way. Perhaps any issues you've come across and what solutions you've found. I guess just a general thread about using Bleachbit on the RasPi. 
I'd also like to say thank you to Andrew for answering my question on UbuntuForums so many many years ago and thanks to everyone that makes this awesome program possible!

**Comment by az0** [comment]
I have not tried it on Raspberry Pi. BleachBit requires Python, which seems to run well on RP. For command line interface, BleachBit does not require PyGObject, GTK, X, or Wayland. In other words, it can run headless.

Besides dependencies, the next issue is the location of files. As you are using Debian, locations may be compatible with Debian for X64 systems.

If you run a particular issue, feel free to bring it up.

>I'd also like to say thank you to Andrew for answering my question on UbuntuForums so many many years ago and thanks to everyone that makes this awesome program possible!

You're welcome!
